scripts/tasks/src/swc.ts

```diff
@@ -0,0 +1,43 @@
+import type { Options } from '@swc/core';
+import { logger } from 'just-scripts';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { exec } from 'just-scripts-utils';
+
+function swcCli(options: Options) {
+  const { outputPath, module } = options;
+
+  const cmd = `npx swc ${outputPath} --out-dir ${outputPath} --config module.type=${module?.type}`;
```

Comment:
This runs `swc` transpile on the `lib` and/or `lib-commonjs` folders transpiled by TS